docs/modules/superset/pages/usage-guide/security.adoc

```diff
@@ -126,6 +126,71 @@ Further information for specifying an AuthenticationClass for an OIDC provider c
 Superset has a concept called `Roles` which allows you to grant user permissions based on roles.
 Have a look at the {superset-security}[Superset documentation on Security^{external-link-icon}^].
 
+=== [[OPA]] OPA role mapping
```

Comment:
optional: I personally find that easier to read.
The suggestion also changes it to lower-case, *should* hopefully be fine
```suggestion
[opa]
=== OPA role mapping
```